scripts/debian-libindy/install-indy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+INDY_LIB_VERSION=1.16.0
+UBUNTU_VERSION=bionic
+
+sudo apt-get update && \
+    sudo apt-get install -y software-properties-common apt-transport-https && \
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 68DB5E88 && \
+    sudo add-apt-repository "deb https://repo.sovrin.org/sdk/deb $UBUNTU_VERSION stable" && \
```

Comment:
This is nitpicking, but because I noticed and remembered this rule: "When referencing a variable, it is generally advisable to enclose its name in double-quotes." -- https://tldp.org/LDP/abs/html/quotingvar.html

Reply:
Thanks for the reminder!